README.md

```diff
@@ -0,0 +1,93 @@
+# www-site-checker
+
+This project comprises tooling designed to check on the state of the mozilla.org site.
+
+_CURRENT STATUS: WIP, pre-production_
+
+Current supported checks:
+
+* Outbound URLs from www.mozilla.org - verified by domain and or an allow-list
+
+Roadmap / future checks
+
+* Support throttling, if necessary in real use
+* Site-wide smoke test: confirmation of 200 OK from every single page (in dev/stage/prod)
+* HTML validation of all CMS-authored pages
+
+
+## Usage
+
+### Automatic usage
+Currently, checks are set to run when updated code in this repo is pushed or (TO BE RE-ENABLED) on a twice-a-day schedule. If any issues are spotted, alerts are sent to Sentry and Slack (if either is configured) and report files are uploaded as artefacts as a result of the Github Action (more on this below)
+
+### Manual usage and local development
+You can also clone this repo to your local machine and run it there. This is the recommended approach for developing new checks:
+
+```
+$ git clone git@github.com:mozmeao/www-site-checker.git
+$ cd www-site-checker
+$ python bin/run_checks.py --sitemap-url=https://www.mozilla.org/sitemap.xml
+```
```

Comment:
Perhaps add a note about installing the `requirements.txt` here?

Reply:
🤦 indeed!